cmd/root.go

```diff
@@ -76,10 +70,25 @@ func (a *MethodAzure) InitRootCommand() {
 				return err
 			}
 			if subscriptionID == "" {
-				return errors.New("flag subscription-id is not set")
+				fmt.Print("Warning - flag subscription-id is not set; several commands require this flag to be set\n")
```

Comment:
Are we validating this at the command level now? If so, I think we maybe just remove this entirely